airbyte-integrations/connectors/destination-clickhouse-v2/src/test/kotlin/io/airbyte/integrations/destination/clickhouse_v2/client/ClickhouseSqlGeneratorTest.kt

```diff
@@ -60,6 +61,20 @@ class ClickhouseSqlGeneratorTest {
         }
     }
 
+    @Test
```

Comment:
I added this but it is not testing a lot of logic because there is none. I'm leaving it present so we avoid not writing a test if this method is changing.